internal/pkg/vm/state/testing.go

```diff
@@ -124,10 +115,19 @@ func (m *MockStateTree) GetActorCode(c cid.Cid, protocol uint64) (dispatch.Execu
 // making a test implementation of the cbor store that can map test cids to test
 // states.
 func TreeFromString(t *testing.T, s string, cst *hamt.CborIpldStore) Tree {
-	tree := NewEmptyStateTree(cst)
+	tree := NewTree(cst)
 	strAddr, err := address.NewActorAddress([]byte(s))
 	require.NoError(t, err)
 	err = tree.SetActor(context.Background(), strAddr, &actor.Actor{})
 	require.NoError(t, err)
 	return tree
 }
+
+// MustGetActor gets the actor or panics if it can't.
+func MustGetActor(st Tree, a address.Address) *actor.Actor {
```

Comment:
If `MustGetActor` needs to move, `MustSetActor` should too.

Reply:
got scrambled on an intermediate refactor, MustGetActor and MustSetActor are both here.